package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "danger",
-  "version": "0.14.1",
+  "version": "0.14.2",
```

Comment:
Could you revert this change please? Unless we are good to ship this in a patch release - cc @orta